src/Components/Publishing/Display/DisplayTargeting.tsx

```diff
@@ -0,0 +1,10 @@
+import { data as sd } from "sharify"
+
+export const targetingData = (id: string, pageType: string) => {
+  const isTesting = sd.HASHTAG_LAB_ADS_ENABLED || "true"
+  return {
+    is_testing: JSON.parse(isTesting),
```

Comment:
So there's no divergance between the value you're returning from the function and what's referenced internal to it, best to keep things 1:1: 

```js
const isTesting = JSON.parse(sd.HASHTAG_LAB_ADS_ENABLED || "true")
return {
  is_testing: isTesting,
  ... 
}
```